schema.sql

```diff
@@ -1,5 +1,52 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id              BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY  NOT NULL,
+    name            VARCHAR(255)                         NOT NULL,
+    date_of_birth   DATE                                 NOT NULL,
+    escape_attempts INT                                  NOT NULL,
+    neutered        BOOLEAN                              NOT NULL,
+    weight_kg       DECIMAL                              NOT NULL
 );
+
+/*
+ * Add a column species of type string to your animals table
+*/
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+
+/*
+ * Create a table named owners with the following columns:
+    - id: integer (set it as autoincremented PRIMARY KEY)
+    - full_name: string
+    - age: integer
+*/
+
+CREATE TABLE owners (
+    owner_id              BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY  NOT NULL,
+    full_name       VARCHAR(255)                         NOT NULL,
+    age             INT                                  NOT NULL
+);
```

Comment:
Well done 🙌 You added a new table with the name `owners` and the propper columns 💪